packages/react/src/experimental/OneDataCollection/Filters/FilterTypes/InFilter/InFilter.tsx

```diff
@@ -193,8 +194,8 @@ export function InFilter<T extends string>({
                 )
               }}
             >
-              <span className="line-clamp-1 w-fit text-left">
-                {option.label}
+              <span className="max-w-[250px] flex-1 whitespace-nowrap">
```

Comment:
I tend to use _rem_ for widths because it grows with font size, making more space for text. Not sure if it is applicable here, but it could be a nice polishing touch